packages/nodes-base/nodes/KoboToolbox/KoboToolboxTrigger.node.ts

```diff
@@ -0,0 +1,158 @@
+import {
+	IDataObject,
+	IHookFunctions,
+	INodeType,
+	INodeTypeDescription,
+	IWebhookFunctions,
+	IWebhookResponseData,
+	LoggerProxy as Logger,
+} from 'n8n-workflow';
+
+import {
+	formatSubmission,
+	koboToolboxApiRequest,
+	parseStringList
+} from './GenericFunctions';
+
+import {
+	formattingOptions,
+} from './descriptions';
+
+export class KoboToolboxTrigger implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'KoboToolbox Trigger',
+		name: 'koboToolboxTrigger',
+		icon: 'file:koboToolbox.svg',
+		group: ['trigger'],
+		version: 1,
+		description: 'Process KoboToolbox submissions',
+		defaults: {
+			name: 'KoboToolbox Trigger',
+			color: '#64C0FF',
+		},
+		inputs: [],
+		outputs: ['main'],
+		credentials: [
+			{
+				name: 'koboToolboxApi',
+				required: true,
+			},
+		],
+		webhooks: [
+			{
+				name: 'default',
+				httpMethod: 'POST',
+				responseMode: 'onReceived',
+				path: 'webhook',
+			},
+		],
+		properties: [
+			{
```

Comment:
And another field called "Trigger On". It's a type option and the only value would be On Form Submittion.